doc/Doxyfile

```diff
@@ -879,7 +865,7 @@ IGNORE_PREFIX          =
 # If the GENERATE_HTML tag is set to YES (the default) Doxygen will
 # generate HTML output.
 
-GENERATE_HTML          = YES
```

Comment:
@abergeron told me IRL that this is a 2 step process, another step after will generate the html.